### DISCUSSION TITLE
When using a numerical list, the line spacing will become smaller and the font size will become larger.

### DISCUSSION THREAD

**Discussion by jackie-weiwei** [fleather-editor/fleather#350] [General]
When using a number list, the line spacing will become smaller and the font size will become larger. I don't want this problem to occur. Is there any way to set the line spacing and font size in the number list?

**Comment by amantoux** [comment]
Have you tried on the latest release?

https://github.com/fleather-editor/fleather/releases/tag/v1.14.5%2B1

**Reply by jackie-weiwei** [reply]
I have tried the latest version and the problem still exists
![image](https://github.com/fleather-editor/fleather/assets/101157197/e12a530a-1f21-4a4b-bc8f-a7ab2efecada)

You can see that the upper three lines do not use numerical serial numbers, and the line spacing is very large after line breaks. The lower three lines use numerical serial numbers. The text is obviously larger than that without serial numbers, and the serial number 1 is smaller than 2 and 3.

The following is the document code
[{"insert":"\n\n你好\n你好\n你好\n\n你好"},{"insert":"\n","attributes":{"block":"ol"}},{"insert":"你好"},{"insert":"\n","attributes":{"block":"ol"}},{"insert":"你好"},{"insert":"\n","attributes":{"block":"ol"}}]

**Reply by jackie-weiwei** [reply]
Unordered list also has the same problem
